### PR TITLE
Completed dom-creation exercise - Justin Song

### PR DESCRIPTION
**Pull request by justinsong1608**
Screenshot 2023-02-08 at 3 06 33 PM

I zoomed out to take a screenshot of the whole page.

### CODE REVIEW

File: dom-creation/main.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  var $newHThree = document.createElement('h3');
+  var $newTextTwo = document.createTextNode('#' + pokemon.number);
+  $newHThree.appendChild($newTextTwo);
+  $newDivThree.appendChild($newHThree);
+
+  var $newP = document.createElement('p');
+  var $newTextP = document.createTextNode(pokemon.description);
+  $newP.appendChild($newTextP);
+  $newDivThree.appendChild($newP);
+  return $row.appendChild($newDivOne);
```
Comment (shawnkost) [inline dom-creation/main.js]: You should not be appending to `$row` inside this function. You should only be returning the `column-third` div. Make sure you read all of the instructions correctly

File: dom-creation/main.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
+var $row = document.querySelector('.row');
+for (var i = 0; i < pokedex.length; i++) {
+  renderPokemon(pokedex[i]);
```
Comment (shawnkost) [inline dom-creation/main.js]: Append should happen here

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes requested here!

**Comment by justinsong1608** [comment]
Updated!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**